.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  node_version: "12.x"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ env.node_version }}
+      - name: Cache node_modules
+        uses: actions/cache@v1
+        with:
+          path: node_modules
+          key: ${{ runner.OS }}-node_modules-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+              ${{ runner.OS }}-node_modules-
+      - name: Install JS dependencies
+        run: ls package-lock.json && npm install || yarn install
+      - name: Build
+        run: npm run build
+      - name: Commit and push distified version to master
+        uses: stefanzweifel/git-auto-commit-action@v4.1.1
+        with:
+          commit_message: 'Update distified version'
+          commit_options: '--no-verify'
+          branch: 'master'
```

Comment:
Refer to [`v3.0.0`](https://github.com/stefanzweifel/git-auto-commit-action/releases/tag/v3.0.0) for why it's no longer required to pass the `GITHUB_TOKEN` environment variable.

Reply:
This version also mentions the `branch` input param has become optional although I'm not entirely sure what `git push origin` does without specifying the branch.

Reply:
☝️ https://stackoverflow.com/a/33320981/9599137

So it pushes to the current branch by default. AFAICT this means we can safely remove `branch`.

Reply:
One more thing: https://github.com/stefanzweifel/git-auto-commit-action/releases/tag/v4.0.0

> Tools like husky can now be executed before a commit is created.

Does it indicate the `commit_options` param can also be removed alongside `branch`?

Reply:
> Does it indicate the commit_options param can also be removed alongside branch?

I don't know, feel free to try :)

Reply:
Yep, it does work: https://github.com/c-hive/gha-remove-artifacts/actions/runs/57356983

But we need to ignore `dist/index.js` from being scanned by ESlint, otherwise, it fails.